_maps/map_files/TgStation/tgstation.2.1.3.dmm

```diff
@@ -1164,17 +1164,9 @@
 	},
 /area/ai_monitored/security/armory)
 "acN" = (
-/obj/structure/rack,
-/obj/item/weapon/storage/box/rubbershot,
-/obj/item/weapon/storage/box/rubbershot,
-/obj/item/weapon/storage/box/rubbershot,
-/obj/item/weapon/storage/box/rubbershot,
-/obj/item/weapon/storage/box/rubbershot,
-/obj/item/weapon/storage/box/rubbershot,
-/turf/open/floor/plasteel/bot{
-	dir = 2
-	},
-/area/ai_monitored/security/armory)
+/obj/structure/chair/stool/bar,
+/turf/open/floor/plasteel/bar,
+/area/crew_quarters/bar)
```

Comment:
its because of how tgm works, its cause something else was changed to use acN after the time you used it for you change so it overwrites it